www/class/centreonLDAP.class.php

```diff
@@ -1110,8 +1110,10 @@ protected function updateLdapServers($arId)
      */
     public function setGeneralOptions($arId, $options)
     {
+        $isUpdate = !empty($arId);
```

Comment:
```suggestion
        $isUpdate = ((int) $arId !== 0);
```

Reply:
Indeed, it's weird but ar_id equals 0 on creation